docs/mkdocs.yml

```diff
@@ -0,0 +1,45 @@
+# Project information
+site_name: Gluu Flex 1.0 Docs
+site_description: Documentation for Gluu Flex
+
+# Copyright
+copyright: Copyright &copy; 2022, Gluu, Inc.
+
+# Repository
+docs_dir: source
+repo_name: GluuFederation/flex
+repo_url: https://github.com/GluuFederation/flex/
+edit_uri: edit/4.3/docs/source
```

Comment:
this should be modified. The source should be directing to the docs folder inside the repo

Reply:
I did correct it to the `main` branch. The `source` folder is inside the docs folder inside the repo. This is standard for Mkdocs, the pathing is relative to mkdocs.yml. The `source` folder, as well as an optional `theme` override folder go here, which Mkdocs pulls the .md files from.